### DISCUSSION TITLE
Program not Parsing Keras Metadata

### DISCUSSION THREAD

**Discussion by rpassaro** [kyleskom/NBA-Machine-Learning-Sports-Betting#392] [Q&A] [answered]
Hi, I just tried running the program. I followed step by step, installing all packages, gathering data again and retraining the model, and I'm getting an issue when I actually run it. It says it can't parse the keras metadata. I've attached a screenshot below. Any help would be greatly appreciated. Thanks!
<img width="710" alt="AI" src="https://github.com/kyleskom/NBA-Machine-Learning-Sports-Betting/assets/61886601/77af51f7-e26e-46cb-b99a-335b9a57eaa8">

**Comment by willisj318** [comment, marked as answer]
Just setup and run it in WSL.

**Reply by rpassaro** [reply]
Thanks!
